web/app/js/components/StatusTable.jsx

```diff
@@ -43,20 +44,20 @@ const columns = {
     return {
       title: "Deployment",
       dataIndex: "name",
-      key: "name",
-      render: name => shouldLink ? <ConduitLink to={`/deployment?deploy=${name}`}>{name}</ConduitLink> : name
+      render: (_, row) => (<React.Fragment>
```

Comment:
you can avoid having to do `render: (_, row) =>` by removing the `dataIndex: "name"` above, then you can just do `render: row =>`. You will need to specify a `key: "name"` though.